### PR TITLE
Finished the german translation

### PR DESCRIPTION
**Pull request by JohnnyJTH**
Finished the German translation. It was nearly done but I added the last ones.

### CODE REVIEW

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 1]
```diff
@@ -1,1403 +1,1405 @@
...
+          <source>Refresh (F5)</source>
+          <target state="translated">Aktualisieren (F5)</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
+          <note from="MultilingualEditor" annotates="source" priority="4">It is accurate</note>
+        </trans-unit>
+        <trans-unit id="NavNewItemButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
+          <source>Create a new item (Ctrl + Shift + N)</source>
+          <target state="translated">Neue Datei erstellen (Strg + Umschalt + N)</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
+          <note from="MultilingualEditor" annotates="source" priority="4">It is accurate</note>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.de-DE.xlf:463]: Is it needed?

File: Files/MultilingualResources/Files.de-DE.xlf [hunk 2]
```diff
@@ -1,1403 +1,1405 @@
...
+        </trans-unit>
+        <trans-unit id="NavUpButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
+          <source>Up (Alt + Up Arrow)</source>
+          <target state="translated">Hoch (Alt + Pfeil Hoch)</target>
+        </trans-unit>
+        <trans-unit id="NavRefreshButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
+          <source>Refresh (F5)</source>
+          <target state="translated">Aktualisieren (F5)</target>
+          <note from="MultilingualUpdate" annotates="source" priority="2">Please verify the translation’s accuracy as the source string was updated after it was translated.</note>
+          <note from="MultilingualEditor" annotates="source" priority="4">It is accurate</note>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.de-DE.xlf:457]: Is it needed?

**Review by tsvietOK — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.de-DE.xlf (hunk 2 above)
Comment (JohnnyJTH) [reply Files/MultilingualResources/Files.de-DE.xlf:457]: > Is it needed?

What do you mean?

Comment (tsvietOK) [inline Files/MultilingualResources/Files.de-DE.xlf:457]: If translation has been verified, these both notes should be removed.

File: Files/MultilingualResources/Files.de-DE.xlf (hunk 1 above)
Comment (tsvietOK) [reply Files/MultilingualResources/Files.de-DE.xlf:463]: If translation has been verified, these both notes should be removed.

**Review by tsvietOK — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.de-DE.xlf (hunk 2 above)
Comment (tsvietOK) [reply Files/MultilingualResources/Files.de-DE.xlf:457]: And resolve conflicts, please.

**Pull request closed without merging**